ui/src/components/startpage/Startpage.js

```diff
@@ -10,6 +10,10 @@ class Startpage extends Component {
     spacing: "10"
   };
 
+  componentDidMount(){
+    document.title = "Home | o2r Deno Server"
```

Comment:
This demonstrates why it is important to change the same string in one location only: "Demo Server" vs. "Demoserver" is inconsistent.